docs/contributor/04-10-technical-design.md

```diff
@@ -2,162 +2,110 @@
 
 ## Kyma Istio Operator
 
-![Kyma IstioOperator Overview](../assets/istio-operator-overview.svg)
+The Kyma Istio Operator implements one controller that consists of several self-contained reconciliation components. The logic is split up in reconciliation components to 
+have a better extensibility and maintainability. This means each of these components must have its clearly separated responsibility and must work in isolation when assessing whether reconciliation is required, applying changes, and returning a status.
+To understand the reasons for the technical design of the Kyma Istio Operator, refer to the [Architecture Decision Record](https://github.com/kyma-project/istio/issues/135).
 
-We want to keep the Kyma Istio operator as simple as possible. That's why we decided to start with one controller that consists of several self-contained components executing reconciliation logic. The controller uses [Istio CR](../user/03-technical-reference/istio-custom-resource/01-30-istio-custom-resource.md) as a resource, which must be present in the `kyma-system` Namespace. It only reconciles the oldest Istio CR in the `kyma-system` Namespace, based on its creation timestamp. After this reconciliation process is completed, all other Istio CRs will be in the `error` state.
+The following diagram illustrates the Kyma Istio Operator and its components:
 
-### Ownership of current resources in Kyma repository
+![Kyma Istio Operator Overview](../assets/istio-operator-overview.svg)
 
-In order to transition to a more modularised architecture, the [IstioOperator resource](https://github.com/kyma-project/kyma/tree/main/resources/istio),
-the [additional istio-resources](https://github.com/kyma-project/kyma/tree/main/resources/istio-resources), and
-the [certificates](https://github.com/kyma-project/kyma/tree/main/resources/certificates) must be moved to the new modules.
+## Istio CR
 
-#### Kyma Istio Operator resource
+The [Istio CR](../user/03-technical-reference/istio-custom-resource/01-30-istio-custom-resource.md) a namespace-scoped resource that is used to manage the Istio installation. The reason for Istio CR being namespace-scoped is that it was created and used before the Kyma Istio Operator was introduced.
+There is no advantage to the Istio CR being namespace-scoped as the Kyma Istio Operator only supports one Istio CR per cluster. However, it is not possible to change it to cluster-scoped without breaking changes.
 
-The Kyma Istio Operator resource is moved into the new Kyma Istio Operator. It is used to define default values for Istio, which the user can customise by modyfying Istio CR.
+If multiple Istio CRs exist on the cluster, the Kyma Istio operator will only use the oldest Istio CR for reconciliation and put the other Istio CRs in an error state.
```

Comment:
Should we also mention that it needs to be in `kyma-system` and the recommended name is `default`?

Reply:
Hmm, good point. I will think about what's the best way to add it to the design, because the defaults should also be mentioned in a more customer facing documentation and I can't find this. So I will check where we need to add this as well.

Reply:
https://github.com/kyma-project/istio/pull/495/commits/8ca3eefd59acb3167b8ccd8150a020780c0ec3ea

Reply:
I added it: https://github.com/kyma-project/istio/pull/495/commits/8ca3eefd59acb3167b8ccd8150a020780c0ec3ea